WordPress/Classes/ViewRelated/People/InvitePersonViewController.swift

```diff
@@ -0,0 +1,393 @@
+import Foundation
+import UIKit
+import WordPressShared
+import SVProgressHUD
+
+
+
+/// Allows the user to Invite Followers / Users
+///
+class InvitePersonViewController : UITableViewController {
+
+    // MARK: - Public Properties
+
+    /// Target Blog
+    ///
+    var blog: Blog!
+
+
+    // MARK: - Private Properties
+
+    /// Invitation Username / Email
+    ///
+    private var usernameOrEmail: String? {
+        didSet {
+            refreshUsernameCell()
+            validateInvitation()
+        }
+    }
+
+    /// Invitation Role
+    ///
+    private var role : Role = .Follower {
+        didSet {
+            refreshRoleCell()
+            validateInvitation()
+        }
+    }
+
+    /// Invitation Message
+    ///
+    private var message : String? {
+        didSet {
+            refreshMessageTextView()
+
+            // Note: This is a workaround. For some reason, the textView's properties are getting reset
+            setupMessageTextView()
+        }
+    }
+
+    /// Last Section Index
+    ///
+    private var lastSectionIndex : Int {
+        return tableView.numberOfSections - 1
+    }
+
+    /// Last Section Footer Text
+    ///
+    private let lastSectionFooter = NSLocalizedString("Add a custom message (optional).", comment: "Invite Footer Text")
```

Comment:
Maybe rename this to `lastSectionFooterText` or `lastSectionFooterTitle` to be a bit more descriptive of it returning a string.